go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/crossplane/crossplane
 
-go 1.17
+go 1.18
 
 require (
```

Comment:
We can update `crossplane-runtime` version, too. It'd be great if you can run Crossplane with that change and see if it crashes or has anything unexpected in the logs 🙂